dcos/config.py

```diff
@@ -160,12 +161,17 @@ def _enforce_config_permissions(path):
     :type path: str
     :rtype: None
     """
-    permissions = oct(stat.S_IMODE(os.lstat(path).st_mode))
-    if permissions not in ['0o600', '0600']:
-        msg = ("Permissions '{}' for configuration file '{}' are too open. "
-               "File must only be accessible by owner. "
-               "Aborting...".format(permissions, path))
-        raise DCOSException(msg)
+
+    if sys.platform == 'win32':
+        return
```

Comment:
Let's write a comment as to why we are doing this. We can point to the GitHub issue too.